### DISCUSSION TITLE
Released demo dataset (71 trajectories) appears smaller than the dataset described in the paper

### DISCUSSION THREAD

**Discussion by pandaliza** [UT-Austin-RobIn/oopsieverse#64] [Q&A]
Thanks for releasing the code and demos — the DamageSim plugin has been straightforward to get running.

While setting up the IL experiments I found that the Hugging Face release is smaller than what the paper describes, and I wanted to check whether I'm looking in the wrong place.

**What's in `ut-robin-lab/oopsieverse-demos`:** 54 HDF5 files, 71 trajectories total. 37 files contain a single `demo_0`; 17 contain `demo_0` and `demo_1`.

- `behavior1k/teleop`: 26 files, 27 demos (only `shelve_item_safe.hdf5` has 2)
- `robocasa/teleop`: 28 files, 44 demos

**What the paper describes:** Sec. IV states 90 demonstrations across five tasks (Shelve Cereal Box, Lift Egg, Add Firewood, Pour Water, Wipe Countertop), evenly split between the with- and without-health-feedback conditions. The conclusion states 450 demonstrations. Those five tasks are all BEHAVIOR-1K side, where the release has ~11 trajectories combined.

**Questions:**

1. Is the IL training set behind Fig. 5 intended to be released separately, or is the HF repo meant to be the complete dataset? The docs describe these files in the context of `quickstart_*.py` playback, so I may be reading a visualization catalog as a training set.
2. Sec. IV / conclusion disagree on 90 vs. 450 — is one a typo?
3. The paper's manipulation is with vs. without live health feedback, but filenames encode `_safe` / `_unsafe`, which is an outcome label. Is the collection condition stored in per-demo attributes anywhere? Without it the with/without-feedback split isn't recoverable from the release.

**Reproduction** (reads HDF5 metadata over HTTP, no full download):

```python
from huggingface_hub import HfFileSystem
import h5py

fs = HfFileSystem()
files = sorted(fs.glob("datasets/ut-robin-lab/oopsieverse-demos/**/*.hdf5"))
total = 0
for path in files:
    with fs.open(path, "rb") as fobj, h5py.File(fobj, "r") as f:
        n = len(f["data"].keys())
    total += n
    print(f"{path.split('oopsieverse-demos/')[-1]}: {n}")
print("files:", len(files), "demos:", total)
```
